api-dummy.sha1

```diff
@@ -1 +1 @@
-49b90e5f6fec86bf6785e198e9ca65d46a5937c1
+420e0cc99b20c98eb6f415bcf78815c51508dd6f
```

Comment:
should we do a similar version of https://ci--alfred.elifesciences.org/job/dependencies-search-update-api-sdk-php/ for api-dummy and run it weekly?